pkg/diagnostics/component_monitoring.go

```diff
@@ -29,6 +29,7 @@ const (
 	BulkGet                  = "bulk_get"
 	BulkDelete               = "bulk_delete"
 	CryptoOp                 = "crypto_op"
+	JobExecutionOp           = "job_execution_op"
```

Comment:
Thoughts on renaming this to `job_trigger_op` or `job_triggering_op`? I want to keep the same verbiage as we do in [the docs](https://docs.dapr.io/developing-applications/building-blocks/jobs/howto-schedule-and-handle-triggered-jobs/) and we call the 'execution' the job triggering.